src/odemis/acq/stitching/test/tiledacq_test.py

```diff
@@ -55,16 +56,17 @@
 from odemis.util.comp import compute_camera_fov, compute_scanner_fov
 
 logging.getLogger().setLevel(logging.DEBUG)
+logging.basicConfig(format="%(asctime)s  %(levelname)-7s %(module)s:%(lineno)d %(message)s")
 
 CONFIG_PATH = os.path.dirname(odemis.__file__) + "/../../install/linux/usr/share/odemis/"
-ENZEL_CONFIG = CONFIG_PATH + "sim/enzel-sim.odm.yaml"
 METEOR_CONFIG = CONFIG_PATH + "sim/meteor-sim.odm.yaml"
+METEOR_FIBSEM_CONFIG = CONFIG_PATH + "sim/meteor-fibsem-sim.odm.yaml"
 
 
 class CRYOSECOMTestCase(unittest.TestCase):
     @classmethod
     def setUpClass(cls):
-        testing.start_backend(ENZEL_CONFIG)
+        testing.start_backend(METEOR_FIBSEM_CONFIG)
```

Comment:
Maybe a thought for another PR but not this one. Should the class be called SECOMTestCase?

Reply:
Cryo SECOM is the old name for ENZEL. I could rename it "METEORTestCase"... but there is already such test case. So we need to find a better name. I gave up for now, but if you have a good suggestion (for the 2 test cases), I'm happy to adjust the names :-)

Reply:
Maybe CryoAcqTestCase , just one name I can think